### DISCUSSION TITLE
Python issues

### DISCUSSION THREAD

**Discussion by maddykelley** [openearth/aeolis-python#208] [General]
What version of python works with newest AeoLiS update?

**Comment by Sierd** [comment]
We try to stay compatible with the latest python version. We run tests for python 3.8, 3.9 and 3.10. I would advise running 3.10 but I am running 3.11 myself on my installation.

**Reply by cswann00** [reply]
I'm trying to run Aeolis for the first time and having issues with Python 3.12. I'll try downgrading to 3.11 and see if that works. Really excited to run AeoLis on some of my OBX 3D timeseries of dune growth. Happy to share that dataset with the entire group as well. It will be my first time, so I hope I don't get stuck on the learning curve too long !

**Reply by maddykelley** [reply]
We (Christy and I) were able to run the SandMotor model example on her computer using newest aeolis (rc3) and python 3.10. 
Yay!

**Reply by maddykelley** [reply]
@cswann00 here are the scripts I mentioned to help setup your own grid:
https://github.com/openearth/aeolis-python/tree/main/tools/preprocessing

**Reply by Sierd** [reply]
Hi @cswann00, nice to see that things are working. 
Do not hesitate to reach out if you need additional help. 
I am always curious to see how things may work (or not).
